src/main/java/uk/gov/hmcts/reform/sscs/config/TribunalsHearingsEventDeadLetterQueueListener.java

```diff
@@ -0,0 +1,89 @@
+package uk.gov.hmcts.reform.sscs.config;
+
+import com.azure.messaging.servicebus.ServiceBusClientBuilder;
+import com.azure.messaging.servicebus.ServiceBusReceivedMessage;
+import com.azure.messaging.servicebus.ServiceBusReceiverClient;
+import com.azure.messaging.servicebus.models.SubQueue;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.hmcts.reform.sscs.model.hearings.HearingRequest;
+import uk.gov.hmcts.reform.sscs.service.AppInsightsService;
+
+@Slf4j
+@Configuration
+@RequiredArgsConstructor
+@ConditionalOnProperty("flags.tribunals-to-hearings-api.enabled")
+public class TribunalsHearingsEventDeadLetterQueueListener {
+
+    private final TribunalsHearingEventQueueProperties tribunalsHearingEventQueueProperties;
+    private final AppInsightsService appInsightsService;
+    private boolean keepRun = true;
```

Comment:
Optional: Could we rename this to `keepRunning` ?

Reply:
Tbh might remove this going to have a catch up with Sean tomorrow and just run it forever was debating it but seen the other team keep it in so thought why not

Reply:
Resolved